spec/jekyll-remote-theme/theme_spec.rb

```diff
@@ -0,0 +1,90 @@
+RSpec.describe Jekyll::RemoteTheme::Theme do
+  let(:owner) { "foo" }
+  let(:name) { "bar" }
+  let(:nwo) { "foo/bar" }
+  let(:git_ref) { nil }
+  let(:raw_theme) { git_ref ? "#{nwo}@#{git_ref}" : nwo }
+  subject { described_class.new(raw_theme) }
+
+  it "stores the theme" do
+    expect(subject.instance_variable_get("@raw_theme")).to eql(nwo)
+  end
+
+  context "with an abnormal NWO" do
+    let(:nwo) { " FoO/bAr " }
+
+    it "normalizes the nwo" do
+      expect(subject.instance_variable_get("@raw_theme")).to eql("foo/bar")
```

Comment:
Do we know for sure that codeload will respect these kinds of capitalization differences? E.g. my theme is `BenBalter/foOOOO` but we request `benbalter/fooooo`.

Reply:
Yes. I tested and it's case insensitive.